maro/rl/training/train_ops.py

```diff
@@ -108,6 +101,9 @@ def set_policy_state(self, policy_state: object) -> None:
         """
         self._policy.set_state(policy_state)
 
+    def to_device(self, device: str):
+        pass
```

Comment:
I think this method should be implemented by all its subclasses. Therefore, we should add `@abstractmethod` and use `raise NotImplementedError`.